test_conformance/vulkan/vulkan_interop_common/vulkan_utility.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef _vulkan_utility_hpp_
+#define _vulkan_utility_hpp_
+
+#include "vulkan_wrapper_types.hpp"
+#include <vector>
+#include <ostream>
+#include <string.h>
+#include <map>
+
+#define ARRAY_SIZE(a)  (sizeof(a) / sizeof(a[0]))
```

Comment:
This is already provided by the harness

Reply:
Seems like a tiny fix. Will fix it along with build and CI issues.

Reply:
This is fixed in latest revision.